melatonin/components/component_tree_view_item.h

```diff
@@ -48,6 +48,43 @@ namespace melatonin
+                    67,51,51,7,67,154,153,131,67,0,0,0,67,0,0,136,67,0,0,0,67,99,109,0,0,128,67,0,0,112,67,98,0,0,128,67,51,51,103,67,154,153,131,67,0,0,96,67,0,0,136,67,0,0,96,67,108,0,0,152,67,0,0,96,67,98,102,102,156,67,0,0,96,67,0,0,160,67,51,51,103,67,0,0,160,67,0,
+                    0,112,67,108,0,0,160,67,0,0,136,67,98,0,0,160,67,102,102,140,67,102,102,156,67,0,0,144,67,0,0,152,67,0,0,144,67,108,0,0,136,67,0,0,144,67,98,154,153,131,67,0,0,144,67,0,0,128,67,102,102,140,67,0,0,128,67,0,0,136,67,108,0,0,128,67,0,0,112,67,99,109,0,
+                    0,184,67,0,0,0,67,108,0,0,200,67,0,0,0,67,98,102,102,204,67,0,0,0,67,0,0,208,67,51,51,7,67,0,0,208,67,0,0,16,67,108,0,0,208,67,0,0,48,67,98,0,0,208,67,205,204,56,67,102,102,204,67,0,0,64,67,0,0,200,67,0,0,64,67,108,0,0,184,67,0,0,64,67,98,154,153,179,
+                    67,0,0,64,67,0,0,176,67,205,204,56,67,0,0,176,67,0,0,48,67,108,0,0,176,67,0,0,16,67,98,0,0,176,67,51,51,7,67,154,153,179,67,0,0,0,67,0,0,184,67,0,0,0,67,99,109,0,0,176,67,0,0,112,67,98,0,0,176,67,51,51,103,67,154,153,179,67,0,0,96,67,0,0,184,67,0,0,96,
+                    67,108,0,0,200,67,0,0,96,67,98,102,102,204,67,0,0,96,67,0,0,208,67,51,51,103,67,0,0,208,67,0,0,112,67,108,0,0,208,67,0,0,136,67,98,0,0,208,67,102,102,140,67,102,102,204,67,0,0,144,67,0,0,200,67,0,0,144,67,108,0,0,184,67,0,0,144,67,98,154,153,179,67,0,
+                    0,144,67,0,0,176,67,102,102,140,67,0,0,176,67,0,0,136,67,108,0,0,176,67,0,0,112,67,99,109,0,0,232,67,0,0,0,67,108,0,0,248,67,0,0,0,67,98,102,102,252,67,0,0,0,67,0,0,0,68,51,51,7,67,0,0,0,68,0,0,16,67,108,0,0,0,68,0,0,48,67,98,0,0,0,68,205,204,56,67,102,
+                    102,252,67,0,0,64,67,0,0,248,67,0,0,64,67,108,0,0,232,67,0,0,64,67,98,154,153,227,67,0,0,64,67,0,0,224,67,205,204,56,67,0,0,224,67,0,0,48,67,108,0,0,224,67,0,0,16,67,98,0,0,224,67,51,51,7,67,154,153,227,67,0,0,0,67,0,0,232,67,0,0,0,67,99,109,0,0,224,
+                    67,0,0,112,67,98,0,0,224,67,51,51,103,67,154,153,227,67,0,0,96,67,0,0,232,67,0,0,96,67,108,0,0,248,67,0,0,96,67,98,102,102,252,67,0,0,96,67,0,0,0,68,51,51,103,67,0,0,0,68,0,0,112,67,108,0,0,0,68,0,0,136,67,98,0,0,0,68,102,102,140,67,102,102,252,67,0,
+                    0,144,67,0,0,248,67,0,0,144,67,108,0,0,232,67,0,0,144,67,98,154,153,227,67,0,0,144,67,0,0,224,67,102,102,140,67,0,0,224,67,0,0,136,67,108,0,0,224,67,0,0,112,67,99,109,0,0,232,67,0,0,160,67,108,0,0,248,67,0,0,160,67,98,102,102,252,67,0,0,160,67,0,0,0,
+                    68,154,153,163,67,0,0,0,68,0,0,168,67,108,0,0,0,68,0,0,184,67,98,0,0,0,68,102,102,188,67,102,102,252,67,0,0,192,67,0,0,248,67,0,0,192,67,108,0,0,232,67,0,0,192,67,98,154,153,227,67,0,0,192,67,0,0,224,67,102,102,188,67,0,0,224,67,0,0,184,67,108,0,0,224,
+                    67,0,0,168,67,98,0,0,224,67,154,153,163,67,154,153,227,67,0,0,160,67,0,0,232,67,0,0,160,67,99,101,0,0 };
+
```

Comment:
Loving this idea but is it reliable for you? When I click back to the inspector, keyboard focus is lost, making it feel  glitchy. Wondering if we can pepper around `setMouseClickGrabsKeyboardFocus(false)` to solve (or have the icon also show "last focus").

Also, I'm missing seeing wantsKeyboardFocus in the hierarchy, so I guess I'd put in an less bright icon for that and move the icons to the end of the label unless you object.